### PR TITLE
Added Sight Machine to Adopters List

### PR DESCRIPTION
**Pull request by xamox**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind documentation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area integrations

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

No issues, just adding to adopters list.

```release-note
NONE
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -10,3 +10,5 @@ This is a list of production adopters of Falco (in alphabetical order):
...
 
 * [Sumo Logic](https://www.sumologic.com/) - Sumo Logic provides a SaaS based log aggregation service that provides dashboards and applications to easily identify and analyze problems in your application and infrastructure. Sumo Logic provides native integrations for many CNCF projects, such as Falco, that allows end users to easily collect Falco events and analyze Falco events on DecSecOps focused dashboards.
+
+* [Sight Machine](https://www.sightmachine.com) - Sight Machine is the category leader for manufacturing analytics and used by Global 500 companies to make better, faster decisions about their operations. Sight Machine uses Falco to help enforce SOC2 compliance as well as a tool for real time security monitoring and alerting in kubernetes.
```
Comment (jonpulsifer) [inline ADOPTERS.md]: Sight Machine should be placed above Sumo Logic as this list is in alphabetical order

Comment (xamox) [reply ADOPTERS.md]: Sorry about that, will fix.

Comment (xamox) [reply ADOPTERS.md]: Done

**Comment by leodido** [comment]
@xamox thanks! LGTM  🎉 

Nevertheless could you please rebase it on top of the current develop? Thanks :)

File: ADOPTERS.md (hunk 1 above)
Comment (fntlnz) [reply ADOPTERS.md]: Thanks!

**Comment by xamox** [comment]
> @xamox thanks! LGTM 
> 
> Nevertheless could you please rebase it on top of the current `dev` branch? Thanks :)

Alright, merged latest dev and did a rebase.  Let me know if I need anything else.

**Comment by fntlnz** [comment]
Uhm, there's something wrong in the commit history :thinking:  `This branch cannot be rebased due to conflicts`

**Pull request closed without merging**